docs/docs/constraints.md

```diff
@@ -30,20 +30,21 @@ Regex is allowed for LIKE and UNLIKE operators; to match ANY value, use the stri
 
 ### CLUSTER operator
 **Value** (optional): A string value.
-When a value is specified then tasks are launched on agent nodes with an attribute whose value matches exactly.
+When a value is specified, tasks are launched on agent nodes with an attribute whose value matches exactly.
 
 `CLUSTER` allows you to run all of your app's tasks on agent nodes that share a certain attribute.
-This is useful for example if you have apps with special hardware needs, or if you want to run them on the same rack for low latency.
+This is useful, for example, if you have apps with special hardware needs, or if you want to run them on the same rack for low latency.
 
-A field of `"hostname"` tells Marathon that launched tasks of the app/pod have affinity for each other and should be launched together on the same agent:
-* When value is specified then tasks are launched on the agent whose hostname matches the value.
-* When value is empty or unspecified then the first instance is launched on **any** agent node and the remaining tasks are launched alongside it on the same agent.
+The <hostname> field tells Marathon that the launched tasks of the app or pod should be launched together on the same agent:
+* When a value for <hostname> is specified, tasks are launched on the agent whose hostname matches the value.
+* WWhen the <hostname> value is empty or unspecified, the first instance is launched on **any** agent node and the remaining tasks are launched alongside it on the same agent.
 
 Attribute fields are handled differently:
-* When value is specified then tasks are launched on any agent with an attribute named according the field **and** with a value matching that of the constraint.
-* When value is empty or unspecified then the first instance is launched on any agent with an attribute named according to the field; the value of the attribute on that agent is used for future constraint matches.
+* When attribute fields are specified, tasks are launched on any agent with an attribute named according the field **and** with a value matching that of the constraint.
```

Comment:
I really did mean `value` and not `attribute`. "When a value is specified, tasks are launched ..."

Reply:
Maybe this is confusing because "Hostname field" and "Attribute field" are bad section names? Maybe a better way to explain it would be:
```
FIELD := "hostname" | ATTRIBUTE      // FIELD is the literal "hostname" or else some ATTRIBUTE
ATTRIBUTE := {some mesos agent attribute name}
```